### ISSUE TITLE
Prevent webRTC leaks in the chrome browser

### ISSUE THREAD

**Issue by petrpatek** [apify/fingerprint-suite#328]
**Describe the feature**
A feature to prevent WebRTC leaks in Chrome, specifically to secure real/local IP information.

**Motivation**
For web scraping, preventing WebRTC leaks is essential. Blocking IP leaks through WebRTC helps maintain the scraper’s anonymity, even when using VPNs or proxies, reducing the risk of detection and IP exposure.

**Constraints**
- Ensure that this feature does not interfere with WebRTC functionality in contexts where it’s intentionally enabled.
- Maintain minimal impact on browser performance during scraping sessions.

**Proposed solution for Chrome**
I have tested this one in Chrome in the unblocking project I am working on, and it worked. see the attached screenshot. 
```javascript
const script = `
const inject = () => {
    if (window.hasRunWebRTCControl) return;
    window.hasRunWebRTCControl = true;

    // Override RTCPeerConnection to prevent IP leaks
    const OriginalRTCPeerConnection = window.RTCPeerConnection;
    function RTCPeerConnectionModified(config, constraints) {
        if (config && config.iceServers) config.iceServers = [];
        return new OriginalRTCPeerConnection(config, constraints);
    }
    RTCPeerConnectionModified.prototype = OriginalRTCPeerConnection.prototype;
    window.RTCPeerConnection = RTCPeerConnectionModified;

    // Disable getUserMedia for camera/microphone access
    navigator.mediaDevices.getUserMedia = () => Promise.reject(new Error('getUserMedia is disabled.'));

    console.log("WebRTC protection injected");
};
inject();
`;

// Inject the script at page load
await page.addInitScript(script);
```

**Comment by B4nan** [comment]
I believe we already have something like this, its just not enabled by default, cc @barjin 

https://github.com/apify/fingerprint-suite/blob/d8dbd211fd97e39606eb9cd3dee6ee794861fe76/packages/fingerprint-injector/src/utils.js#L487

**Comment by barjin** [comment]
Yes, you can effectively disable WebRTC in the browser by generating a fingerprint with `mockWebRTC: true` (and injecting it). As far as I remember, the patch creates a Proxy object that catches all function calls and just immediately resolves with `undefined` (and does this recursively) - something like your patch for `navigator.mediaDevices.getUserMedia`.

Can you please verify that `mockWebRTC` works for you? If it broke in some cases, I'm more than happy to switch for your implementation - I always thought the current one was a little bit crude. Cheers!

**Comment by barjin** [comment]
Closing due to inactivity (also, this is likely solved by the `mockWebRTC: true` option).
